Kernel/Devices/Serial/VirtIO/Console.h

```diff
@@ -7,8 +7,8 @@
 
 #pragma once
 
-#include <Kernel/Bus/VirtIO/ConsolePort.h>
 #include <Kernel/Bus/VirtIO/Device.h>
+#include <Kernel/Devices/Serial/VirtIO/ConsolePort.h>
 #include <Kernel/Memory/RingBuffer.h>
 
 namespace Kernel::VirtIO {
```

Comment:
Should we still use the `Kernel::VirtIO` namespace for the console and RNG device, now that you moved them out of the Bus/VirtIO directory?

Reply:
So, I looked at this again and I don't think we should change this. Putting it out is reasonable, but we might want to wait with this until we get the new PCI driver for this device being integrated in the actual refactor PR.